src/data_manager/data_file_manager.py

```diff
@@ -128,61 +147,65 @@ def process_config(self):
         # Create our DataTypeConfig (which in turn create our SubTypeConfig) objects.
         self.dispatch_to_object()
 
-    def _search_submission_data(self, dataType, subEntry):
-
-            subType = subEntry
-
-            try:
-                returned_dict = next(item for item in self.submission_system_data['snapShot']['dataFiles']
-                                     if item['dataType'].get('name') == dataType and item['dataSubType'].get('name') == subType)
-
-            except StopIteration:
-                logger.debug('dataType: %s subType: %s not found in submission system data.' % (dataType, subType))
-                logger.debug('Creating entry with \'None\' path and extracted path.')
-                returned_dict = {
-                    'dataType': dataType,
-                    'subType': subType,
-                    'path': None,
-                    'tempExtractedFile': None
-                }
-
-            return returned_dict
+    def _search_submission_data(self, data_type, sub_type):
+
+        try:
+            returned_dict = next(item for item in self.submission_system_data['snapShot'] \
+                                                                             ['dataFiles']
+                                 if item['dataType'].get('name') == data_type
+                                 and item['dataSubType'].get('name') == sub_type)
+        except StopIteration:
+            self.logger.debug('dataType: %s subType: %s not found in submission system data.',
+                              data_type,
+                              sub_type)
+            self.logger.debug('Creating entry with \'None\' path and extracted path.')
+            returned_dict = {
+                'dataType': data_type,
+                'subType': sub_type,
+                'path': None,
+                'tempExtractedFile': None}
+
+        return returned_dict
 
     def query_submission_system(self):
+        '''get file information from Submission System (FMS)'''
 
-        # The list of tuples below is created to filter out submission system data against our config file.
+        # The list of tuples below is created to filter out submission
+        # system data against our config file.
         ontologies_to_transform = ('GO', 'DOID', 'MI', 'ECOMAP')  # These have non-generic loaders.
 
-        self.transformed_submission_system_data['releaseVersion'] = self.submission_system_data['snapShot']['releaseVersion']['releaseVersion']
+        self.transformed_submission_system_data['releaseVersion'] \
+                = self.submission_system_data['snapShot']['releaseVersion']['releaseVersion']
```

Comment:
I don't know if its a personal preference, but I prefer the equal sign to be on the same line as the subject.  It would be good to discuss this as a standard, or if there is already a widely used standard for this, I'm happy to read up on it.

Reply:
I don't know for sure what the standard is for the equals sign. I know the PEP8 standard is to have the "+" at the beginning of the line though. I naturally prefer it on the left but we could look it up / discuss it.